### ISSUE TITLE
Is this repository still in WIP Mode?

### ISSUE THREAD

**Issue by sumitpore** [jedrzejchalubek/wp-seeder#1]
Hi,

The idea of porting Laravel Seeding to WordPress seems really interesting.

I wanted to use this for one of my projects, however, repository's description says that it is in WIP. Would like to know if it is still in that mode or it is ready to use? 

If it is still in WIP mode, I would like to contribute to make it usable.

**Comment by jedrzejchalubek** [comment]
Yes, It is still in work in progress :)

**Comment by stevewolfe555** [comment]
Did you ever decide to follow on from this project?

**Comment by jedrzejchalubek** [comment]
@stevewolfe555 Yeah, I think this is something missing in the WP world and I think it is a cool idea. However, still missing a time and willingness to push it forward

**Comment by stevewolfe555** [comment]
Interestingly I found another, project called FakerPress which seems to do most of this through its ajax and slow.
